collectors/apps.plugin/apps_plugin.c

```diff
@@ -3806,6 +3852,29 @@ static void send_charts_updates_to_netdata(struct target *root, const char *type
     }
 }
 
+static void send_proc_states_count(usec_t dt)
+{
+    static bool chart_added = false;
+    // create chart for count of processes in different states
+    if (!chart_added) {
+        fprintf(
+                stdout,
+                "CHART system.process_states '' 'Apps Process States' 'numbers' processes system.process_states line %d %d\n",
```

Comment:
```suggestion
                "CHART system.process_states '' 'Apps Process States' 'processes' processes system.process_states line %d %d\n",
```

We usually avoid such generic units. Let's change it to "processes".